python/FlexSource.py

```diff
@@ -129,9 +147,8 @@ def start(self):
         print "FlexSource::Starting..."
         self.radio = FlexApi().getRadio()
 
-        # TODO: make these parameters of source block
-        dax_ch = 1
-        sample_rate = 192000
+        # TODO: make this a parameter of the source block
+        sample_rate = 192000 # Available in IQStream.cs
```

Comment:
Yea, we'll have to remember that there are only a few options for this parameter (I think we have 24, 48, 96 and 192 Khz to work with)